### DISCUSSION TITLE
Bridge between SQLAlchemy ORM Models and Pydantic

### DISCUSSION THREAD

**Discussion by Actticus** [sqlalchemy/sqlalchemy#6608] [Usage Questions] [answered]
I'm trying to migrate from Tortoise ORM to SQLAlchemy and I can't find any normal solution to transform the SQLAlchemy model to Pydantic, for validation and serialization data. Have you any working solution for it?

**Comment by zzzeek** [comment, marked as answer]
pydantics ORM integration is documented at https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances .

**Reply by Actticus** [reply]
Oh, ty, i haven't see that <3
